### ISSUE TITLE
can't install neo4j on ubuntu 14.04 server

### ISSUE THREAD

**Issue by Artgit** [amoswood/puppet-neo4j#3]
I can't install can't neo4j on ubuntu 14.04 server with a following error:

```
==> default: Error: Could not set 'file' on ensure: No such file or directory - /opt/neo4j/neo4j-community-2.1.5/conf/neo4j.properties20141026-1275-14525ma.lock at 179:/tmp/vagrant-puppet-3/modules-0/concat/manifests/init.pp
==> default: Error: Could not set 'file' on ensure: No such file or directory - /opt/neo4j/neo4j-community-2.1.5/conf/neo4j.properties20141026-1275-14525ma.lock at 179:/tmp/vagrant-puppet-3/modules-0/concat/manifests/init.pp
==> default: Wrapped exception:
==> default: No such file or directory - /opt/neo4j/neo4j-community-2.1.5/conf/neo4j.properties20141026-1275-14525ma.lock
==> default: Error: /Stage[main]/Neo4j/Concat[/opt/neo4j/neo4j-community-2.1.5/conf/neo4j.properties]/File[/opt/neo4j/neo4j-community-2.1.5/conf/neo4j.properties]/ensure: change from absent to file failed: Could not set 'file' on ensure: No such file or directory - /opt/neo4j/neo4j-community-2.1.5/conf/neo4j.properties20141026-1275-14525ma.lock at 179:/tmp/vagrant-puppet-3/modules-0/concat/manifests/init.pp
```

I use following virtualboxin my Vagrant script - https://oss-binaries.phusionpassenger.com/vagrant/boxes/latest/ubuntu-14.04-amd64-vbox.box

and Puppet script for provision:

```
class { 'neo4j' :
   version => '2.1.5',
   auth_ensure         => present,
   auth_admin_user     => 'admin',
   auth_admin_password => 'admin',
}
```

I have added my Vagrant/Puppet project here http://www.filedropper.com/ubuntuserver14

What I'm doing wrong ? Thanks!

**Comment by mlambrichs** [comment]
Take a look at:
https://github.com/amoswood/puppet-neo4j/pull/6
Solved it for me.

**Comment by amoswood** [comment]
Merged #6.
